Allow to disable localization.

	# this patch removes need to gettext and intltool
	# but intltool was needed for eautoreconf so I add m4_pattern_allow
	# which is probably not the best way but it's working

Patch by Arfrever Frehtes Taifersar Arahesis

--- configure.in
+++ configure.in
@@ -30,7 +30,11 @@
 AM_PROG_CC_STDC
 AC_HEADER_STDC
 dnl localization tools
-ifdef([IT_PROG_INTLTOOL],[IT_PROG_INTLTOOL],[AC_PROG_INTLTOOL])
+AM_NLS
+if test "$USE_NLS" != "no"; then
+ m4_pattern_allow([AC_PROG_INTLTOOL])
+	ifdef([IT_PROG_INTLTOOL],[IT_PROG_INTLTOOL],[AC_PROG_INTLTOOL])
+fi
 dnl Initialize libtool
 AC_PROG_LIBTOOL
 AM_PROG_LIBTOOL
